cosmos/io.py

```diff
@@ -0,0 +1,183 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import Any
+from urllib.parse import urlparse
+
+from cosmos import settings
+from cosmos.constants import FILE_SCHEME_AIRFLOW_DEFAULT_CONN_ID_MAP
+from cosmos.exceptions import CosmosValueError
+from cosmos.settings import remote_target_path, remote_target_path_conn_id
+
+
+def upload_artifacts_to_aws_s3(project_dir: str, **kwargs: Any) -> None:
```

Comment:
Since the syntax of artifact differs between UK and US (artifact x artefact) WDYT if we just called this method `upload_to_aws_s3`? (same feedback applies to other methods)

Reply:
I have removed that word :). However, I feel we could discuss and standardise on whether to follow UK or US English and stick to that in our docs/naming. WDYT?

Reply:
I agree, @pankajkoti , we currently follow more US English (behaviour x behaviour), but we can certainly improve our consistency in the future. It may be worth avoiding these in the project functions/interfaces when possible.